cmd/build/v2/build_test.go

```diff
@@ -119,11 +119,7 @@ func TestOnlyInjectVolumeMountsInOkteto(t *testing.T) {
 		},
 		CurrentContext: "test",
 	}
-	dir, err := os.MkdirTemp("", "test")
-	if err != nil {
-		t.Fatal(err)
-	}
-	defer os.RemoveAll(dir)
+	dir := t.TempDir()
```

Comment:
Don't we need specific names for the folder to make the test work? Some of these tests don't rely on the folder name for the build context?

Reply:
Not those ones, the ones that needed specific names or be relative to a path I left them as they were